### ISSUE TITLE
Create environment using Anaconda

### ISSUE THREAD

**Issue by shubulan** [ageron/handson-ml#524]
When I tried to create an environment using Anaconda with the following command

```
$ conda env create -f environment.yml
```
I got the following error
```
Collecting package metadata (repodata.json): done
Solving environment: failed

ResolvePackageNotFound:
  - xgboost==0.82
```
How can I fix it？

**Comment by ageron** [comment]
Hi @shubulan ,

Thanks for your question! Make sure you pull the latest version of this project, I removed this line from the environment.yml file, and I just have py-xgboost=0.90 in there, which should work fine.
Are you using Windows?

**Comment by shubulan** [comment]
Yes, I'm using Windows 10 1903. I did use the lastest version, and the file environment.yml still use the line
![image](https://user-images.githubusercontent.com/48401928/70683010-aaf7bf00-1cdb-11ea-8306-5a5ec6a0848b.png)
Maybe it is a typing error？
I create a environment by commenting out this line. I don't know what errors will come out in the future. I will install the package py-xgboost=0.90 separately.

**Comment by ageron** [comment]
Oh sorry, my bad, I thought this was the repository for the 2nd edition, at https://github.com/ageron/handson-ml2. I'll update the environment.yml file in this repo as well. Thanks again for your feedback.
